generated/client/jobs/jobs_client.go

```diff
@@ -180,6 +182,46 @@ func (a *Client) GetJobs(params *GetJobsParams, opts ...ClientOption) (*GetJobsO
 	return nil, runtime.NewAPIError("unexpected success response: content available as default response in error", unexpectedSuccess, unexpectedSuccess.Code())
 }
 
+/*
+GetJobsEvents subscribes to job events e g status updates
```

Comment:
Should read `e.g.`, right?

Reply:
Bug in the swagger genator :) I've rephrased it to avoid the bug